repo2docker/buildpacks/conda/activate-conda.sh

```diff
@@ -1,7 +1,8 @@
 # enable conda and activate the notebook environment
```

Comment:
You can debug activation here with `set -x`. It will very verbose, but you'll be able to see whether that `mamba activate` in the bottom works or not. You could also run `which mamba` there to make sure you are calling the shell function and not the executable. Otherwise replace `mamba activate` with `source $(mamba shell activate)`.